subsys/usb/usb_device.c

```diff
@@ -232,9 +232,11 @@ static bool usb_handle_request(struct usb_setup_packet *setup,
 /*
  * @brief send next chunk of data (possibly 0 bytes) to host
  *
+ * @param [in] rlen        Requested length
+ *
  * @return N/A
  */
-static void usb_data_to_host(u16_t len)
+static void usb_data_to_host(u16_t rlen)
```

Comment:
This does not belong to this patch and not needed IMO

Reply:
I will move it to another PR. But it is necessary. There are many `len`, can you say without much thought what is meant by `len` in the context?

Reply:
At least I know that it is len, rlen is not better here.